### DISCUSSION TITLE
你希望RTS都有哪些功能？

### DISCUSSION THREAD

**Discussion by seven1240** [rts-cn/rts#24] [RTS]
RTS诞生了，欢迎阅读RTS相关文档，告诉我们你期望RTS能帮你实现什么。

- http://rts.cn/docs/RTS/
- http://rts.cn/docs/RTS/faq

**Comment by visitewen** [comment]
视频那块的及录像那块的优化 ^_^

**Reply by seven1240** [reply]
关于这些，XSwitch里有一些优化。RTS里还没有。

**Comment by geyunhu** [comment]
支持通过els控制发送update功能

**Reply by seven1240** [reply]
这是个太小众的需求，在XSwitch的商用版中已支持，暂时没有准备放到RTS中。
